repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/module-setup.sh

```diff
@@ -30,11 +30,17 @@ install() {
     #dracut_install /etc/selinux/*/policy/*
     #dracut_install $(find /etc/selinux/*/contexts)
 
+    # shellcheck disable=SC2154 # Following variables are assigned by dracut
+    { # NOTE: use these new variables instead to avoid the warning
+    local _initdir=$initdir
+    local _moddir=$moddir
+    }
+
     # NOTE: rather remove it then keep it; we really don't want plymouth now
     #       (hopefully forever)
     # remove the plymouth text plugin so we get either graphics or details
-    rm -rf ${initdir}/$(plymouth --get-splash-plugin-path)text.so \
-          ${initdir}/usr/share/plymouth/themes/text/*
+    rm -rf "${_initdir}/$(plymouth --get-splash-plugin-path)text.so" \
+           "${_initdir}/usr/share/plymouth/themes/text/*"
```

Comment:
Are you sure that there will be only 1 file in that folder? Otherwise the added quotes will cause `rm` not to delete anything.

Example:
```
mkdir /tmp/test
cd /tmp/test
touch aa ba
mkdir dir
touch dir/cc
rm -rf "/tmp/test/*"
ls  # All files are still there, since "/tmp/test/*" expands to "/tmp/test/aa /tmp/test/ba /tmp/test/dir" as one string, but there is no such file
```